Algorithms_and_Data_Structures/AI Search Algorithms/Uninformed Search/bfs.py

```
@@ -0,0 +1,33 @@
+def BFS(adj, V, s, visited):
+    queue = []
+    
+    queue.append(s);
```

Ruff (0.12.2)

Statement ends with an unnecessary semicolon

Remove unnecessary semicolon

(E703)